src/Bicep.Cli.IntegrationTests/ProgramTests.cs

```diff
@@ -164,6 +168,7 @@ public void BuildSingleFileToStdOutShouldProduceExpectedTemplate(DataSet dataSet
             {
                 result.Should().Be(0);
                 output.Should().NotBeEmpty();
+                error.Should().MatchRegex(BuildSummarySucceededRegex);
```

Comment:
Looks like our logger implementation logs everything to stderr. @majastrz - I remember you looked into which scenarios we should use stderr/stdout for. Does this fit with your expectations?

Reply:
That's right. Tools that produce output via StdOut must send warnings to StdErr because there's no other stream available. I was shocked by this at the time, so we did follow up with our language mentors and this was the conclusion. Conditionally switching streams also makes things harder for consumers in various shells and CI jobs, so keeping it consistent makes sense.

Reply:
Agreed. Printing any log level to stdout would risk break my usage of the `--stdout` flag in CI tools!